README.md

```
@@ -25,43 +25,43 @@ Whether you're building websites, APIs, microservices, or embedded systems (such
 
 Finch comes packed with essential tools to streamline your development process. Here's a detailed breakdown of its key capabilities:
 
-### 🚀 WebSocket Support
+###  WebSocket Support
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)